src/components/GameCards/GameCard.js

```diff
@@ -32,9 +36,13 @@ export default {
     },
 
     gameCardClass() {
-      const badge = this.gameProgress === '100'
-        ? 'badge'
-        : '';
+      let badge = '';
+
+      if (this.showGameInfoOnCover) {
+        badge = this.gameProgress === '100'
+          ? 'badge'
+          : '';
+      }
```

Comment:
you can avoid this if statement if you merge the two checks:

```
const badge = this.showGameInfoOnCover && this.gameProgress === '100'
    ? 'badge'
    : '';

Reply:
You're right. I don't even know why I added the additional if statement... 😵